ansible_base/authentication/models/authenticator_map.py

```diff
@@ -35,38 +36,49 @@ class Meta:
         default=False,
         help_text=(_("If a user does not meet this rule should we revoke the permission")),
     )
+    map_type_choices = [
+        ('allow', 'allow'),
+        ('is_superuser', 'is_superuser'),
+        ('organization', 'organization'),
+        ('team', 'team'),
+    ]
+    if 'ansible_base.rbac' in settings.INSTALLED_APPS:
+        from ansible_base.rbac.models import RoleDefinition
+
+        role = models.ForeignKey(
+            RoleDefinition, null=True, on_delete=models.SET_NULL, related_name="authenticator_maps", help_text=(_("The role this mapping belongs to"))
```

Comment:
I think I might the help text to something like: `The role this map will grant the authenticating user to the targeted object`